src/Libraries/CoreNodeModelsWpf/Charts/BarChartNodeModel.cs

```diff
@@ -58,11 +58,15 @@ public class BarChartNodeModel : NodeModel
         /// </summary>
         public BarChartNodeModel()
         {
-            InPorts.Add(new PortModel(PortType.Input, this, new PortData("labels", "A list of labels for the bar chart categories.")));
-            InPorts.Add(new PortModel(PortType.Input, this, new PortData("values", "A list (of lists) to supply values for the bars in each category.")));
-            InPorts.Add(new PortModel(PortType.Input, this, new PortData("colors", "A list of colors for each bar chart category.")));
-
-            OutPorts.Add(new PortModel(PortType.Output, this, new PortData("labels:values", "Dictionary containing label:value key-pairs")));
+            InPorts.Add(new PortModel(PortType.Input, this,
+                new PortData("labels", "A list of labels for the bar chart categories.\n\nList<string>")));
```

Comment:
I thought we localize these port tooltips using resource strings, instead of the hardcoded strings?

Reply:
yes, adding ports this was is an anti pattern for internal nodes, attributes should be used!

Reply:
an example to follow:
https://github.com/DynamoDS/Dynamo/blob/master/src/Libraries/CoreNodeModels/ColorRange.cs#L24

Reply:
Sorry, Deyan added these per my request--I didn't know how they were supposed to be coded :(

Reply:
no worries @hwahlstrom we need to do a better job documenting this somewhere and eventually get it implemented for all our internal nodes so that they are localizable. 

I think there is a task, but it's a lot of work.

Reply:
Many thanks for observing this. You learn something new every day. Let's fix this now.

Reply:
Sorry, I should’ve done that without asking to be honest. My fault, will swap over for resource strings in a jiffy!